src/Application/HttpModels/SendRequest.php

```diff
@@ -59,4 +60,9 @@ public function __construct()
         $this->id = (Uuid::uuid4())->toString();
         $this->env = getenv('APP_ENV');
     }
```

Comment:
I'm not 100% sure if the constructor is actually ever run - do you know whether it's run when we do
```
            $sendRequest = $this->serializer->deserialize(
                $this->request->getBody(),
                SendRequest::class,
                'json'
            );
``` in Send?

Reply:
I had certainly assumed so. I think it must be otherwise `Actions\Send` would be crashing when it read `$sendRequest->id`